tests/integration/targets/mso_schema_site_bd_subnet/tasks/main.yml

```diff
@@ -593,7 +660,7 @@
 - name: Verify non_existing_site_bd
   assert:
     that:
-    - non_existing_site_bd.msg == "Provided BD 'ansible_test_2' does not exist. Existing site BDs{{':'}} "
+    - non_existing_site_bd.msg == "Subnet IP '10.1.0.1/16' not found"
```

Comment:
Is this whole test still useful?

Reply:
changed code slightly so resolving this conversation due to test not needed anymore